modules/local/exclusions/main.nf

```diff
@@ -0,0 +1,35 @@
+process EXCLUSIONS {
+    tag "Identify why sample(s) failed to be typed"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python%3A3.12' :
+        'biocontainers/python:3.12' }"
+
+    input:
+    tuple val(meta), path(mikro_file)
+
+    output:
+    path "${meta.id}_untypable.csv",   emit: results
+    path "versions.yml",               emit: versions
+
+    script:
+    def args = task.ext.args ?: ''
+    def species = meta.Species ?: "Unknown"
+    def qc_status = meta.QCStatus ?: "Unknown"
+    def has_file = mikro_file.toString() != "[]" && mikro_file.size() >0 ? "true" : "false"
```

Comment:
`has_file = mikro_file.toString() != "[]"` doesn't seem right to me, but I feel like I'm missing something. Why doesn't the `mikro_file.size() >0 ? "true" : "false"` part alone work?